### ISSUE TITLE
When pick a time with dateTimePicker, it closed but the value in the model does't change. 

### ISSUE THREAD

**Issue by BingBlog** [icebob/vue-form-generator#102]
This is my code:
```
data: {
		model: {
            date:new Date('2017-01-13')
		},
		schema: {
			fields: [
				{
                    type: "dateTimePicker",
                    label: "日期选择",
                    model: "date",
                    dateTimePickerOptions: {
                        format: "YYYY-MM-DD"
                    },
                    onChanged: function(model, newVal, oldVal, field){
                        console.log(model, newVal, oldVal, field);
                    }
                }
			]
		},

		formOptions: {
			validateAfterLoad: true,
			validateAfterChanged: true
		}
	}
```
I found it does't trigger the **onChanged** function. When I input something like  " ", it has trigger the function and change the value in the model. How should I deal with this situation?

**Comment by icebob** [comment]
What's version are you using? 0.6.1 or 2.0.0?

**Comment by icebob** [comment]
It is fixed in #118
